docroot/themes/humsci/su_humsci_theme/patterns/molecules/horizontal-card/horizontal-card.html.twig

```diff
@@ -34,7 +34,7 @@
     {% endif %}
 
     {% if title %}
-      <h3 class="horizontal-card__title">{{ title }}</h3>
+      <div class="horizontal-card__title">{{ title }}</div>
```

Comment:
Why are you changing this from a heading tag? Seems like this would be better for the document outline?

Reply:
I've seen H&S more often than not set the title to be an `h2` tag. Also because we arent the site builders we can't control the structure of the page. with this being an `h3` there _must_ be an `h2` above it.

Reply:
with HTML5 I'm not sure that is true anymore.
https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines#Defining_headings

This is long but full of info and kind of breaks apart what Mozilla says:
https://css-tricks.com/document-outline-dilemma/
 
Then there is this from 2009: https://www.456bereastreet.com/archive/200901/headings_heading_hierarchy_and_document_outlines/
which has the rules

1. Each document should have a logical heading structure that starts at level 1 and doesn’t skip any levels
2. Anything that visually looks like a heading should be marked up as a real heading
3. The title of the document’s main content should be a level 1 heading
4. There should be only one level 1 heading in each document

Finding specs: https://www.w3.org/TR/WCAG20-TECHS/H69.html
> Authors are encouraged to nest headings hierarchically

Doesn't seem to be required but helpful. 

Looks like the best practices are to have an "as hierarchical" structure as possible and to use heading tags instead of other tags with css to show visual importance as screen readers won't pick up on that.

Reply:
I had a chat with jiatyan and she said any skipped headings are a :no_good_man:

Reply:
Is it a navigation problem with the screen readers that gaps create? IE: "Tell me all h2s"?

Reply:
I think so. besides that currently with the way it is like what is on the live site is an `h2` inside an `h3`. definitely not a good thing there

Reply:
Tell me more

Reply:
i cant remember what site it was but they used this pattern in a view. then the title field they configured to render as an h2

Reply:
Aye, I agree. H2s in H3s is very bad but that can be solved with tag stripping. 

This is going to be a large problem for us with products open to general use. I want to make sure we get this right and do the best we can to protect from poor config choices. 

Preventing nested headings is a good idea. Missing out on headings or incorrectly setting heading levels is going to be hard.

Reply:
And H&S would prefer to have "Unstyled" patterns. they would rather add styling via field classes to accomplish their desired outcome.

Reply:
Interesting.

Reply:
Yep, the "unstyled" patterns have been a discussion with them. To what point they want things to be unstyled is variable, too. They want structure and general layout, but maybe not size (dictated by heading tag or otherwise), spacing, etc.

What if our patterns/components never used heading tags by default, but rather they leveraged the display variables (A, B, C, D, etc.) to make a title div look like a heading. So, heading tags are only used in plain, WYSIWYG like content. While this alone wouldn't close the door on users making poor decisions, it would verify that we are providing them with tools that by default may lead to the wrong decision.

Reply:
From what i understand from them they literally want no styling on a pattern.